chat-monitor.js

```diff
@@ -133,30 +146,34 @@ function initConfig() {
 //Checks all config options and loads them appropriately
 function loadSettings() {
     //Add settings wheel to page
-    $( ".ember-chat-container").append("<div id=\"settings-wheel\"> <i class=\"material-icons\">settings</i> </div>");
-    $( "#settings-wheel").click(function() {
-      GM_config.open();
-    });
+    $('.ember-chat-container').append('<div id="settings-wheel"> <i class="material-icons">settings</i> </div>');
+    $('#settings-wheel').click(() => GM_config.open());
```

Comment:
Personally I prefer the more expanded version of this for readability. I'm being nit picky on this because I'm already reviewing this.

Reply:
Runaway nesting levels are a problem with javascript, so if that's what you're talking about, I disagree.